packages/serverless/src/model/global/user.ts

```diff
@@ -95,7 +95,7 @@ export class ModelUser extends ModelGeneral<OptionalId<TUserRecord>> {
     );
     if (!(await collection.isExist())) {
       await collection.index({ userName: 1 }, { unique: true, session });
-      await collection.index({ publicKey: 1 }, { unique: true, session });
+      await collection.index({ publicKey: 1 }, { unique: false, session });
```

Comment:
I think we should remove unique: false, it's false by default

Reply:
fixed